docs/build/js-client/3-setting-up.md

```diff
@@ -1,94 +1,27 @@
 # Setting up project
 
-## Install dependencies
+## CLI install
 
-Initialize an empty `npm` package:
+The best way to initialize the project is by using CLI
 
-```sh
-npm init
-```
-
-In newly created `package.json` file add module entry
-```json
-{
-  "type": "module",
-  ...
-}
-```
-
-We will need JS-client itself
-
-```sh
-npm install @fluencelabs/js-client@unstable
-```
+> You can find an installation guide [here](https://github.com/fluencelabs/cli).
 
-We will need the Fluence CLI to use the Aqua compiler, but only as a development dependency:
+Run the command below and follow instruction.
 
 ```sh
-npm install --save-dev @fluencelabs/cli
+fluence init -t ts --env kras 
 ```
 
-Aqua comes with the standard library, which can be accessed from "@fluencelabs/aqua-lib" package. This package in bundled in Fluence CLI, so you don't need to install anything.
-
-And last, but not least, we need TypeScript:
+Go to `src/frontend` dir and run `npm install`.
 
-```sh
-npm install --save-dev typescript ts-node
-```
-
-Create `tsconfig.json` file and add following content there
-
-```json
-{
-  "compilerOptions": {
-    "target": "ESNext",
-    "module": "ESNext",
-    "strict": true,
-    "skipLibCheck": true,
-    "moduleResolution": "nodenext"
-  }
-}
-```
+Place aqua files in `src/aqua` and compile them via `fluence aqua` command
```

Comment:
what aqua files? i have no idea what that means and looking into the `src/frontend` i was sent, i see no aqua files. all i've done is run npm install so far. 
assuming i was able to find the files, manually copy them (why manually?) and compile them ... then what? next section tells me to delete the frontend part.